addon/components/infinity-loader.js

```diff
@@ -69,5 +73,14 @@ export default Ember.Component.extend({
     if (this.get('infinityModel.reachedInfinity') && this.get('destroyOnInfinity')) {
       this.destroy();
     }
+  }),
+
+  infinityModelPushed: Ember.observer('infinityModel.[]', function() {
```

Comment:
Performancewise it's the same, but slightly different in operation: https://github.com/emberjs/ember.js/issues/2016#issuecomment-14402033

We probably don't want to trigger loadMore if a developer swaps out an item in the array id say